presto-main/src/main/java/io/prestosql/testing/LocalQueryRunnerBuilder.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.testing;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.Session;
+import io.prestosql.spi.session.PropertyMetadata;
+import io.prestosql.spiller.NodeSpillConfig;
+import io.prestosql.sql.analyzer.FeaturesConfig;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+public class LocalQueryRunnerBuilder
+{
+    private Session defaultSession;
+    private FeaturesConfig featuresConfig = new FeaturesConfig();
+    private NodeSpillConfig nodeSpillConfig = new NodeSpillConfig();
+    private boolean initialTransaction;
+    private boolean alwaysRevokeMemory;
+    private Map<String, List<PropertyMetadata<?>>> defaultSessionProperties = ImmutableMap.of();
+    private int nodeCountForStats;
+
+    public static LocalQueryRunner createLocalQueryRunner(Session defaultSession)
```

Comment:
maybe there could be a default session so this parameter would not mandatory

Reply:
`LocalQueryRunner` is main package of `presto-main`, but `io.prestosql.SessionTestUtils#TEST_SESSION` is in testing package. This can be cleaned up later